include/slang/util/CommandLine.h

```diff
@@ -226,6 +226,8 @@ class CommandLine {
     /// @return true on success, false if an errors occurs.
     bool parse(int argc, const char* const argv[]);
 
+    std::string setIgnoreCommand(string_view value);
```

Comment:
When I recommended the set* names for these I was thinking of a slightly different implementation; now that I see what you have here I would recommend these be add* instead because you can call them multiple times and their effects add to each other.

Reply:
ok, I'll take care of that.